opm/material/densead/Evaluation.hpp

```diff
@@ -51,19 +52,28 @@ template <class ValueT, int numVars>
 class Evaluation
 {
 public:
+    //! field type
     typedef ValueT ValueType;
 
-    enum { size = numVars };
+    //! number of derivatives
+    static constexpr int size = numVars;
```

Comment:
I agree, what about simply exposing it as numVars (possibly renaming the template argument numVarsArg or something) or numDerivs?